doc/tutorials/basic/pn_panel.md

```diff
@@ -0,0 +1,440 @@
+# Display Content with `pn.panel`
+
+In this guide, we will learn to display Python objects easily and dynamically with `pn.panel`:
+
+- Display any Python object via `pn.panel(the_object, ...)`.
+
+:::{note}
+When we ask to *run the code* in the sections below, we may execute the code directly in the Panel docs via the green *run* button, in a cell in a notebook, or in a file `app.py` that is served with `panel serve app.py --autoreload`.
+:::
+
+```{pyodide}
+import panel as pn
+pn.extension("plotly", "vega")
+```
+
+## Install the Dependencies
+
+Please make sure [Altair](https://altair-viz.github.io/), [hvPlot](https://hvplot.holoviz.org/index.html), [Matplotlib](https://matplotlib.org/), and [Plotly](https://plotly.com/python/) are installed.
+
+::::{tab-set}
+
+:::{tab-item} conda
+:sync: conda
+
+```bash
+conda install -y -c conda-forge altair hvplot matplotlib plotly
```

Comment:
> I'd also put `pip` first.

I also prefer `pip` first. I put `conda` first because that is what was in the *Getting Started*. Though that was on purpose.

![image](https://github.com/holoviz/panel/assets/42288570/21d71bf8-8084-4d19-a084-c86f9ff09f35)

@philippjfr . Do you agree we should put `pip first`?

Reply:
> I'd prefer the tutorial to start with a section that explains how to get set up, i.e. packages to install and data to download. It could also explain how we expect people to run the tutorial (from the site, a notebook, an editor).

I can change it if more people support your argument Maxime. @philippjfr ?

I would argue that there is some value in keeping it as is.

- You don't have one big up front installation that will be very slow to do for 100 users at an event.
- You can jump right into any section and use it without doing the big up front installation.

Reply:
> You don't have one big up front installation that will be very slow to do for 100 users at an event.

When giving a tutorial I can't imagine asking people to run `pip install` commands here and there, it's just too likely to break and will disrupt people's learning flow. I'd start the tutorial by getting them setup, giving them a lock file or a well defined `requirements.txt`.

Reply:
Yes, let's go with `pip` first and I'd also be fine shipping `requirements.txt` and `environment.yaml` files for the tutorial. In fact the question is, should users copy paste, or should we allow them to download the materials as notebooks?

Reply:
My guess is that they cannot download a lot of the tutorials in the current state. As soon as you start to use dropdowns, tabs etc I thank that possibility is lots. I agree that downloading notebooks and opening in pyodide would be helpful.

Reply:
If you install the myst extension for JLab dropdowns, tabs and admonitions will work just fine:

<img width="1082" alt="Screenshot 2024-02-21 at 16 22 12" src="https://github.com/holoviz/panel/assets/1550771/9cde8be6-f96a-4f4d-94c9-99b469ae8f9b">

Reply:
Indeed, we should immediately add that for our JupyterLite distribution.